docs/devGuide/development/migratingToTypeScript.md

```diff
@@ -149,13 +149,18 @@ Combining the two steps into one - that is, adapting files immediately after ren
 
 The solution to this is to make the rename and adapt commits separate, in order for Git to recognize first that the file has changed into `.ts`, and so the changes are compared not against the old `.js` file, but to the renamed `.ts` file instead.
 
-### Why do we need a normal merge commit
+### Why do we need a rebase and merge commit
 
 Even if the migration developer has kept the history intact through the separate "Rename" and "Adapt" commits, this is only intact at their working branch. At the end of the day, the totality of the changes in the working branch is compared against the target branch, in which the same consideration would apply.
 
 If we do the usual squash commit, the changes from the two commits are combined into a new commit and only that commit will be pushed into the target branch. The original two commits are omitted, therefore the history of the working branch that we have tried to keep intact is stripped away.
 
-The normal merge commit also creates another commit (the merge commit), but the merge process interleaves the commits from the working branch to the target branch, therefore retaining the change history.
+The reason we use a rebase operation is because it updates the base branch of a
```

Comment:
I think the last sentence is fine. I think there's no need to define what a rebase is.